Blog_post

TharunPasam opened the pull request.

lintfail committed:
changes to content

TharunPasam committed:
Changes to Content

TharunPasam committed:
changes to content

TharunPasam committed:
changes to content

TharunPasam committed:
changes to content

TharunPasam committed:
changes to content

lintfail commented on src/content/authors/mark-dinn.md:
```diff
@@ -1,11 +1,11 @@
 ---
-title: Mark Dinn
+title: Tharun Pasam
 image: /images/authors/mark-dinn.jpg
```
Chnage the name to "Survey Sort Team"
Change the image to some random image from upstash

lintfail commented on src/content/authors/mark-dinn.md:
```diff
@@ -1,11 +1,11 @@
+  instagram: https://www.instagram.com
 ---
 
 lorem ipsum dolor sit amet, consectetur adipiscing elit, sed do eiusmod tempor incididunt ut labore et dolore magna aliqua. Ut enim ad minim veniam, quis nostr navigation et dolore magna aliqua.
```
Need to rewrite this

lintfail commented on src/content/posts/post-1.md:
```diff
@@ -1,22 +1,194 @@
-tags: ["diy", "toy"]
+date: 2024-08-01T05:00:00Z
+categories: [Education]
+authors: [Tharun Pasam]
```
Update author name

lintfail commented on src/content/posts/post-2.md:
```diff
@@ -1,5 +1,5 @@
 ---
-title: Why a balloon is a flexible bag
```
What about this post? Delete all posts that are already here, other than the one you created

TharunPasam commented:
-Changed the author name to survey sort team everywhere, and added an image
for the author.
-Deleted remaining posts.
- Related to social media, which accounts should i need to give ?

On Sun, 4 Aug 2024 at 22:48, lintfail ***@***.***> wrote:

> ***@***.**** commented on this pull request.
> ------------------------------
>
> In src/content/posts/post-2.md
> <https://github.com/lintfail/bookworm-light-astro/pull/1#discussion_r1703308403>
> :
>
> > @@ -1,5 +1,5 @@
>  ---
> -title: Why a balloon is a flexible bag
>
> What about this post? Delete all posts that are already here, other than
> the one you created
>
> —
> Reply to this email directly, view it on GitHub
> <https://github.com/lintfail/bookworm-light-astro/pull/1#pullrequestreview-2217788968>,
> or unsubscribe
> <https://github.com/notifications/unsubscribe-auth/BCDXVH5K4SDEY74AW4GHKZDZPZO5NAVCNFSM6AAAAABL2GMRZWVHI2DSMVQWIX3LMV43YUDVNRWFEZLROVSXG5CSMV3GSZLXHMZDEMJXG44DQOJWHA>
> .
> You are receiving this because you authored the thread.Message ID:
> ***@***.***>
>
